api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -1,14 +1,26 @@
 package org.pmiops.workbench.api;
 
 import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```

Comment:
The engineering team collectively agreed that we should use the com.google.common.truth.Truth assertion lib instead of org.junit.jupiter.api.Assertions. Probably ok to keep this assertion. I'm not positive but I don't think google Truth supports this.